pkg/backends/jmeter/backend.go

```diff
@@ -214,6 +216,7 @@ func (b *Backend) Sync(ctx context.Context, loadTest loadTestV1.LoadTest, report
 			return err
 		}
 
+		stats.Record(ctx, observability.MCreatedLoadtestCountStat.M(1))
```

Comment:
why not add this to `controller` instead of to every backend?

Reply:
Good catch, it's now incremented as new namespaces are created (f7c3d63)